### ISSUE TITLE
Migrate to Unibeautify as core

### ISSUE THREAD

**Issue by Glavin001** [Glavin001/atom-beautify#1174]
See https://github.com/Unibeautify/unibeautify

**Comment by MattRiddell** [comment]
This one on the back burner?

**Comment by Glavin001** [comment]
Yeah, I only have so much time so I'm spending it updating the Atom-Beautify code. Once things on Atom-Beautify are going well I will port over the new features to Unibeautify and polish up. 😃

**Comment by Glavin001** [comment]
**Update:** While I am working on the Unibeautify core, @szeck87 has been making great strides in upgrading Atom-Beautify to using Unibeautify.

**Comment by kLOsk** [comment]
Any chance this is still happening? :-/

**Comment by kLOsk** [comment]
Thanks @Glavin001, is it advised / possible to run the current dev branch as an Atom package?

**Comment by szeck87** [comment]
@kLOsk you can clone the repository locally and checkout the `unibeautify` branch.  Then `apm link` it from the terminal/command prompt.  What is there now is pretty out of date though, let me open a PR to merge what I have up to it.

**Comment by michaelprescott** [comment]
FYI, the poll on the atom-beautify page looks like a clickable image of poll results. Visitors that roll over it may expect to click on the image to be taken to the actual poll widget to also give their feedback; however, it's actually an inline widget where each bar is clickable (there are only a few pixels between each item so you can only tell if you roll over the container very slowly).  It's a subtle UI issue, but enough that someone may not realize that clicking anywhere will immediately casts a vote for a feature.  As a result, I imagine the top item is getting very skewed, a cloud based code formatting service. Please, no.

**Comment by jack1142** [comment]
I didn't even know it's clickable and thought they are poll results :o

**Comment by Glavin001** [comment]
Thanks for the feedback, @michaelprescott @jack1142  . I think https://github.com/apex/gh-polls would be the appropriate place for comments on the poll UI itself.

On our end, a message could be added to the [README](https://github.com/Glavin001/atom-beautify#poll-improving-installation-of-third-party-beautifiers) to make it clear the numbers shown are in fact the only poll results view and clicking the bars will cast a vote for the respective option. I would be happy to merge a PR making sure a change 😃 . Thanks in advance!

**Comment by sliterok** [comment]
3 years has passed since initial start of this idea and it hasn't been done yeat. :(

**Comment by stevenzeck** [comment]
I started a new job recently and have not had much time to contribute here.  If you would like to try out what is out there, please look at https://github.com/stevenzeck/atom-beautify/tree/unibeautify for instructions.  It works, but there are gaps between it and the current one, and probably a few bugs too.

**Comment by mgodeneche** [comment]
Hello ! 
Is the migration still a goal or is it cancelled ?

**Comment by Glavin001** [comment]
What features are you most looking forward to? Will help me prioritize which to develop first.

I plan to pick up on maintenance of Atom-Beautify and Unibeautify soon as my work load reduces. Appreciate everyone's patience.

**Comment by ealtunyay** [comment]
I arrived here looking into Liquid template formatting support for Atom

**Comment by UziTech** [comment]
@stevenzeck does the unibeautify branch only work for javascript?

**Comment by nkalpakis21** [comment]
i see this has been open for awhile. is there trouble getting a dev to fix this issue? what is blocking this for so long

**Comment by Novocastrian** [comment]
> i see this has been open for awhile. is there trouble getting a dev to fix this issue? what is blocking this for so long

Probably because Atom is dead: https://github.blog/news-insights/product-news/sunsetting-atom/
